app/controllers/api/abstract_controller.rb

```diff
@@ -17,6 +17,7 @@ class OnlyJson < Exception; end
 
     respond_to :json
     before_action :raw_json, only: [:update, :create]
+    before_action :maybe_enforce_row_lock, only: [:update]
```

Comment:
This line can be removed if it is discovered that row locking creates problems in production. So far, everything looks OK.